app/src/main/java/com/dsciitp/shabd/Setting/SettingFragment.java

```diff
@@ -10,25 +11,30 @@
 
 import com.dsciitp.shabd.R;
 
-import java.util.zip.Inflater;
-
 /**
  * A simple {@link Fragment} subclass.
  */
 public class SettingFragment extends Fragment {
 
 
-    public SettingFragment() {
-        // Required empty public constructor
```

Comment:
Why you removed the required constructor?

Reply:
please review